packages/headless/src/api/search/date/relative-date.ts

```diff
@@ -134,20 +153,12 @@ export function isRelativeDate(date: unknown): date is RelativeDate {
 }
 
 /**
- * Deserializes relative date string value into a `RelativeDate` object.
- * Throws an error if the format is invalid.
```

Comment:
This doesn't throw anymore, if we call `validateRelativeDate` it would be a circular dependency, we would have to extract the meat in a local equivalent method